diff --git a/Code/PgSQL/rdkit/adapter.cpp b/Code/PgSQL/rdkit/adapter.cpp
index aab0284e..6cba4742 100644
--- a/Code/PgSQL/rdkit/adapter.cpp
+++ b/Code/PgSQL/rdkit/adapter.cpp
@@ -63,10 +63,10 @@
 #include <RDGeneral/BoostEndInclude.h>
 
 #ifdef RDK_BUILD_INCHI_SUPPORT
-#include <INCHI-API/inchi.h>
+#include <GraphMol/inchi.h>
 #endif
 #ifdef RDK_BUILD_AVALON_SUPPORT
-#include <AvalonTools/AvalonTools.h>
+#include <GraphMol/AvalonTools.h>
 #endif
 #include <GraphMol/ChemReactions/ReactionFingerprints.h>
 #include <GraphMol/ChemReactions/ReactionUtils.h>
